src/styles/components/header.scss

```diff
@@ -1,2 +1,86 @@
+@import '../mixins';
+
 .header {
```

Comment:
currently only "Software" has a hover effect, all menu items should have this same effect for consistency

Reply:
Perfect. Will make the change now.